snapcraft/plugins/kbuild.py

```diff
@@ -43,6 +43,12 @@
       definitions.  If you don't want default for one or more implicit configs
       coming out of these, just add them to this list as well.
 
+    - build-attributes
+      (list of strings)
+      list of boolean build options:
```

Comment:
we can improve this text in future PRs as this is a core attribute that the plugin understands instead of being an attribute of the plugin itself.